config/routes.rb

```diff
@@ -992,6 +993,11 @@
       get '/help' => 'pages#help', as: :help
       get '/spec.yml' => 'openapi#spec', as: :spec
     end
+
+    namespace :data_api_docs, path: '/data-api' do
```

Comment:
Nothing blocking (as I need to rebase against it 😂) but I was wondering if it'd be good to include "docs" in the path to keep it consistent within this namespace
```suggestion
    namespace :data_api_docs, path: '/data-api-docs' do
```

Reply:
Not sure — merging, can revisit later!